core/src/scss/utilities/variables/core/_colors.scss

```diff
@@ -12,289 +12,612 @@
+//
+$su-color-chocolate: #2f2424 !default;
+
+//
+// <del>$color-chocolate</del>
+//
+// <strong>Deprecated. Will be removed in 6.0.0.</strong><br/>
+// Use <code>$su-color-chocolate</code> instead.
 //
 // Styleguide: Variables.Core.Colors.color-chocolate
 //
-$color-chocolate: #2f2424 !default;
+$color-chocolate: $su-color-chocolate !default;
 
 //
-// $color-clay
+// $su-color-clay
+//
+// Markup: $su-color-clay: #5f574f !default;
 //
-// Markup: $color-clay: #5f574f !default;
+// Styleguide: Variables.Core.Colors.su-color-clay
+//
+$su-color-clay: #5f574f !default;
+
+
+//
+// <del>$color-clay</del>
+//
+// <strong>Deprecated. Will be removed in 6.0.0.</strong><br/>
+// Use <code>$su-color-clay</code> instead.
 //
 // Styleguide: Variables.Core.Colors.color-clay
 //
-$color-clay: #5f574f !default;
+$color-clay: $su-color-clay !default;
+
+//
+// $su-color-cloud
+//
+// Markup: $su-color-cloud: #dad7cb !default;
+//
+// Styleguide: Variables.Core.Colors.su-color-cloud
+//
+$su-color-cloud: #dad7cb !default;
 
 //
-// $color-cloud
+// <del>$color-cloud</del>
 //
-// Markup: $color-cloud: #dad7cb !default;
+// <strong>Deprecated. Will be removed in 6.0.0.</strong><br/>
+// Use <code>$su-color-cloud</code> instead.
 //
 // Styleguide: Variables.Core.Colors.color-cloud
 //
-$color-cloud: #dad7cb !default;
+$color-cloud: $su-color-cloud !default;
 
 //
-// $color-dark-red
+// $su-color-dark-red
 //
-// Markup: $color-dark-red: #820000 !default;
+// Markup: $su-color-dark-red: #820000 !default;
+//
+// Styleguide: Variables.Core.Colors.su-color-dark-red
+//
+$su-color-dark-red: #820000 !default;
+
+//
+// <del>$color-dark-red</del>
+//
+// <strong>Deprecated. Will be removed in 6.0.0.</strong><br/>
+// Use <code>$su-color-dark-red</code> instead.
 //
 // Styleguide: Variables.Core.Colors.color-dark-red
 //
-$color-dark-red: #820000 !default;
+$color-dark-red: $su-color-dark-red !default;
+
+//
+// $su-color-driftwood
+//
+// Markup: $su-color-driftwood: #b6b1a9 !default;
+//
+// Styleguide: Variables.Core.Colors.su-color-driftwood
+//
+$su-color-driftwood: #b6b1a9 !default;
 
 //
-// $color-driftwood
+// <del>$color-driftwood</del>
 //
-// Markup: $color-driftwood: #b6b1a9 !default;
+// <strong>Deprecated. Will be removed in 6.0.0.</strong><br/>
+// Use <code>$su-color-driftwood</code> instead.
 //
 // Styleguide: Variables.Core.Colors.color-driftwood
 //
-$color-driftwood: #b6b1a9 !default;
+$color-driftwood: $su-color-driftwood !default;
 
 //
-// $color-fog
+// $su-color-fog
 //
-// Markup: $color-fog: #f4f4f4 !default;
+// Markup: $su-color-fog: #f4f4f4 !default;
+//
+// Styleguide: Variables.Core.Colors.su-color-fog
+//
+$su-color-fog: #f4f4f4 !default;
+
+
+//
+// <del>$color-fog</del>
+//
+// <strong>Deprecated. Will be removed in 6.0.0.</strong><br/>
+// Use <code>$su-color-fog</code> instead.
 //
 // Styleguide: Variables.Core.Colors.color-fog
 //
-$color-fog: #f4f4f4 !default;
+$su-color-fog: #f4f4f4 !default;
```

Comment:
```suggestion
$color-fog: $su-color-fog !default;
```

Reply:
Missed this one - I'm testing with our Wordpress theme.

Reply:
@cjwest , if you could please commit this one, then I'll do a final check and we can merge 😄

Reply:
@cjwest  - one more thing, I believe this branch is behind so we'll probably need to merge master into it also.